Paintroid/src/main/java/org/catrobat/paintroid/presenter/MainActivityPresenter.java

```diff
@@ -112,9 +112,9 @@
 	private boolean focusAfterRecreate = true;
 
 	public MainActivityPresenter(MainView view, Model model, Navigator navigator, Interactor interactor,
-			TopBarViewHolder topBarViewHolder, BottomBarViewHolder bottomBarViewHolder,
-			DrawerLayoutViewHolder drawerLayoutViewHolder,
-			NavigationDrawerViewHolder navigationDrawerViewHolder, CommandManager commandManager) {
+								TopBarViewHolder topBarViewHolder, BottomBarViewHolder bottomBarViewHolder,
```

Comment:
Please revert unrelated style fixes to `MainActivityPresenter`

Reply:
I have reverted changes, for some reason Android Studio did this. Should I rename branch to only PAINTROID-14 and make a new request? I tought this solved the problem bacuse I tried going to the Import image from gallery ->  Hamburger menu -> Pocket paint and it no longer chrashes even though it did for play store version. Maybe this is another bug.

Reply:
I will have another look at it, maybe this does fix our crashes already.